doc/switch.txt

```diff
@@ -374,8 +374,13 @@ Ruby ~
 
 Hash style (g:switch_builtins.ruby_hash_style):
 >
-    foo = { :one => 'two' }
-    foo = { one: 'two' }
+    foo = { :one => 'two', :three => 'four'  }
+    foo = { one: 'two', :three => 'four'  }
+<
+Oneline Hash (g:switch_builtins.ruby_oneline_hash):
+>
+    foo = { :one => 'two', :three => 'four' }
+    foo = { one: 'two', three: 'four' }
```

Comment:
To make it clearer which happens when, it would be nice to explain where the cursor should be for these two cases. Maybe something like:

```
Hash style (g:switch_builtins.ruby_hash_style): 
>
    foo = { :one => 'two', :three => 'four'  }
    foo = { one: 'two', :three => 'four'  }
<
Cursor: in the hash, on a single hash element.

Oneline Hash (g:switch_builtins.ruby_oneline_hash):
>
    foo = { :one => 'two', :three => 'four' }
    foo = { one: 'two', three: 'four' }
<
Cursor: in the hash, not on any one element -- opening `{` would work.
```

Same for the erb case below, I guess. Does that make sense?

Reply:
Sure! The position of the cursor has not been specified elsewhere, but it makes sense here.